pkg/plugins/platform/local.go

```diff
@@ -423,14 +473,12 @@ func (l *LocalApp) Start() error {
 	// Write docker-compose.yaml (if it doesn't exist).
 	// If the user went through the `ddev config` process it will be written already, but
 	// we also do it here in the case of a manually created `.ddev/config.yaml` file.
-	if !fileutil.FileExists(l.AppConfig.DockerComposeYAMLPath()) {
-		err := l.AppConfig.WriteDockerComposeConfig()
-		if err != nil {
-			return err
-		}
+	err := l.AppConfig.WriteDockerComposeConfig()
```

Comment:
are we going to 'own' the compose file with this? We may want to consider using the file signature solution randy implemented here if so.